<launch>

	<include file="$(find serial)/launch/serial.launch"/>
	<include file="$(find multiwar)/launch/multiwar.launch"/>

	<node pkg="camera" type="camera_node"  name="camera_node"  output="screen" />
	<node pkg="detect" type="armor_detect_node"  name="armor_detect_node"  output="screen" />
<!-->
	<node pkg="recoder" type="recoder_node"  name="recoding"  output="screen" />
	-->
	<node pkg="decision" type="decision_node"  name="decision_node"  output="screen" />

</launch>